extractor/src/main/scala/com/digitalasset/extractor/Extractor.scala

```diff
@@ -214,7 +214,8 @@ class Extractor[T <: Target](config: ExtractorConfig, target: T) {
         config.appId,
         LedgerIdRequirement(ledgerId = "", enabled = false),
         CommandClientConfiguration(1, 1, overrideTtl = true, java.time.Duration.ofSeconds(20L)),
-        sslContext = config.tlsConfig.client
+        sslContext = config.tlsConfig.client,
+        None
```

Comment:
Add `parameterName = None` here. Usually named arguments at the end of the arguments list are optional arguments and, given that they can be avoided selectively, it is safer and cleaner to always name them.